php/Blockonomics.php

```diff
@@ -361,11 +362,30 @@ public function testSetup()
         }
         return $test_results;
     }
+
+    public function settings_test_setup($api_key, $btc_enabled, $bch_enabled)
```

Comment:
Looks like we are still passing the settings values to test setup:
```
$api_key = isset($_GET["api_key"]) ? sanitize_text_field(wp_unslash($_GET['api_key'])) : "";
$btc_active = isset($_GET["btc_active"]) ? $_GET["btc_active"] : false;
$bch_active = isset($_GET["bch_active"]) ? $_GET["bch_active"] : false;
```
We want to instead fetch these from the saved settings, as we were doing before: https://github.com/blockonomics/woocommerce-plugin/blob/master/php/Blockonomics.php#L45-L58

`public function settings_test_setup($api_key, $btc_enabled, $bch_enabled)` should be `public function settings_test_setup()`

`$this->test_one_crypto($code, $api_key);` should be `$this->test_one_crypto($code);` etc.

We want to avoid passing the api_key, btc_active, bch_active as function params. These values should be collected from the saved settings values.